v2/samples/insights/v1api/v1api20210501preview_diagnosticsetting.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: insights.azure.com/v1api20210501preview
```

Comment:
You'll need to make a recording of the sample - see [our docs](https://azure.github.io/azure-service-operator/contributing/add-a-new-code-generated-resource/#run-test-for-added-sample-and-commit-the-recording) for how to do this. 

We have this step to guarantee that our samples all work - most of us have had the experience of trying out a sample and finding out it doesn't work because of a typo, or because it's old, that sort of thing. This helps us to avoid that.

Reply:
Added the recording of the sample.